### PR TITLE
Add OpenInfra final version

### PR DESCRIPTION
**Pull request by JhengLu**
#280

### CODE REVIEW

**Comment by JhengLu** [comment]
#280

File: source/_data/SymbioticLab.bib [hunk 1]
```diff
@@ -1903,3 +1903,18 @@ @InProceedings{dpack:eurosys25
...
 } 
+
+@InProceedings{openinfra:hotinfra24,
+  author = {Jiaheng Lu and Yunming Xiao and Shmeelok Chakraborty and Silvery Fu and Yoon Sung Ji and Ang Chen and Mosharaf Chowdhury and Nalini Rao and Sylvia Ratnasamy and Xinyu Wang},
+  title         = {OpenInfra: A Co-simulation Framework for the Infrastructure Nexus},
```
Comment (mosharaf) [inline source/_data/SymbioticLab.bib:1909]: OpenInfra -> {OpenInfra}

**Comment by mosharaf** [comment]
@JhengLu

**Review by mosharaf — APPROVED** [review]

File: source/_data/SymbioticLab.bib (hunk 1 above)
Comment (JhengLu) [reply source/_data/SymbioticLab.bib]: It is resolved now.

**Pull request merged**